### PR TITLE
wails: complete private key import method

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are adding private key import method on wails.

### CODE REVIEW

File: wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx [hunk 1]
```diff
@@ -64,39 +66,28 @@ const PromptNickname = () => {
...
-            </div>
-            <div className="min-w-fit w-full">
-              <Button type="submit">Next</Button>
-            </div>
-          </div>
-        </form>
-      </div>
-    </div>
+    <Layout>
+      <Stepper step={1} steps={['Private Key', 'Username', 'Password']} />
```
Comment (Thykof) [inline wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx:70]:
```suggestion
      <Stepper step={1} steps={['Private Key', 'Account name', 'Password']} />
```

File: wails-frontend/src/pages/ImportPrivateKey/PromptPrivateKey.tsx [hunk 2]
```diff
@@ -52,37 +54,28 @@ const PromptPrivateKey = () => {
...
-            </div>
-            <div className="min-w-fit w-full">
-              <Button type="submit">Next</Button>
-            </div>
-          </div>
-        </form>
-      </div>
-    </div>
+    <Layout>
+      <Stepper step={0} steps={['Private Key', 'Username', 'Password']} />
```
Comment (Thykof) [inline wails-frontend/src/pages/ImportPrivateKey/PromptPrivateKey.tsx:58]:
```suggestion
      <Stepper step={0} steps={['Private Key', 'Account name', 'Password']} />
```

File: wails-frontend/src/pages/newPassword.tsx [hunk 3]
```diff
@@ -95,43 +96,43 @@ function NewPassword() {
   }
 
   return (
-    <div className="bg-primary min-h-screen">
+    <Layout>
+      {isImportAction && (
+        <div className="pb-6">
+          <Stepper step={2} steps={['Private Key', 'Username', 'Password']} />
```
Comment (Thykof) [inline wails-frontend/src/pages/newPassword.tsx:102]:
```suggestion
          <Stepper step={2} steps={['Private Key', 'Account name', 'Password']} />
```

**Review by Thykof — CHANGES_REQUESTED** [review]

File: wails-frontend/src/pages/ImportPrivateKey/PromptPrivateKey.tsx [hunk 4]
```diff
@@ -52,37 +54,28 @@ const PromptPrivateKey = () => {
...
+      <form ref={form} onSubmit={handleSubmit}>
+        <h1 className="mas-title pt-6">{req.Msg}</h1>
+        <p className="mas-body pt-4">Enter your Private Key</p>
+        <div className="pt-4">
+          <Password
+            defaultValue=""
+            name="privateKey"
+            placeholder={'Private key'}
```
Comment (mazmassa) [inline wails-frontend/src/pages/ImportPrivateKey/PromptPrivateKey.tsx:66]:
```suggestion
            placeholder="Private key"
```

File: wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx (hunk 1 above)
Comment (Thykof) [reply wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx:70]: i18n is not in wails yet

File: wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx [hunk 5]
```diff
@@ -64,39 +66,28 @@ const PromptNickname = () => {
...
+            name="nickname"
+            placeholder={'Account name'}
+            error={error?.nickname}
+          />
+        </div>
+        <div className="flex gap-4 pt-4">
+          <Button variant={'secondary'} onClick={handleCancel}>
+            Cancel
+          </Button>
+          <Button type="submit">Next</Button>
```
Comment (mazmassa) [inline wails-frontend/src/pages/ImportPrivateKey/PromptNickname.tsx:86]: Any reason that we are not using `Intl.t("key...")` in the page!?

**Review by mazmassa — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**